app/jobs/ingest_job.rb

```diff
@@ -0,0 +1,17 @@
+class IngestJob < Hyrax::ApplicationJob
+  queue_as Hyrax.config.ingest_queue_name
+
+  after_perform do |job|
+    opts = job.arguments.reverse.detect { |x| x.is_a? Hash } || {}
```

Comment:
@atz OK. Could you add that as a comment above this line? That's helpful to know.